Sources/SwiftyTailwind/Downloader.swift

```diff
@@ -44,6 +50,10 @@ class Downloader: Downloading {
         return try! localFileSystem.tempDirectory.appending(component: "SwiftyTailwind")
     }
     
+    static let sha256FileName: String = "sha256sums.txt"
+    static let checksumValidator: ChecksumValidating = ChecksumValidation()
+    var numRetries = 0
```

Comment:
I'd not store state in the instance. Since the function is calling itself recursively for retrying, I'd suggest to include that as one extra argument in the function.

Reply:
Done!